web/packages/plugins/shared/common/src/components/UiPluginSelector.tsx

```diff
@@ -224,6 +225,7 @@ export function UIPluginSelector(props: {
           onOpen={onOpen}
           categories={categories}
           config={config}
+          type={entity.type}
```

Comment:
Type should be removed from the UiPlugin interface.
As a requirement is that it passes an entity which MUST have a type

Reply:
okey, removed